driver/web/docs/resources/bbs/accounts.yaml

```diff
@@ -41,6 +41,14 @@ post:
       explode: false
       schema:
         type: integer
+    - name: range_index
```

Comment:
I think the name `range_index` is somewhat unclear. I would suggest something like `page_start_id` or something similar to make it clear that this value is the ID of the item that marks the start of the next page (first id not included in the next page). In the description we should also indicate that this should be the `id` of the last item that was retrieved on the previous page.